### DISCUSSION TITLE
How to update color values in `_color.scss` or `_palette.scss`

### DISCUSSION THREAD

**Discussion by AidaIssayeva** [razonyang/hugo-theme-bootstrap#189] [Q&A] [answered]
I've noticed an interesting thing: whenever I tried to update the value of `hbs-primary` color in `custom.css`, upon rendering it gets overwritten by the values in `_color.scss` and `_palette.scss`(quick swap of the color happens). 
When I try to update values in `_color.scss` - nothing happens: not in incognito mode, not with disabling fast render, nothing. 

After some digging, I found out localhost is not watching for updates in `src` folder, where `_color.scss` is located:
```
Watching for changes in /Users/name/Sites/blog/{archetypes,assets,content,i18n,layouts,package.json,static}
Watching for config changes in /Users/name/Sites/blog/config/_default
```

Is there any way to include `src` folder along with others during the `hugo server -D` or is there a different way of updating values of color parameters?

**Comment by razonyang** [comment, marked as answer]
Hugo won't watch the files placed in `src`, since I use other tools(npm) for generating JS and CSS files.

You can change the color by modifying **assets/css/custom.css** with the following style:

```
.mode-light, .mode-dark {
  --hbs-primary: red !important;
  --hbs-primary-light: green !important;
}
```

`--hbs-primary` and `--hbs-primary-light` were used in **light** and **dark** mode respectively.

**Reply by AidaIssayeva** [reply]
@razonyang it worked! thank you so much! Unfortunately, having issues on Github Web as it doesn't let me mark the answer as accepted 🤷
